GageStatsDB/GageStatsDB.csproj

```diff
@@ -1,20 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>    
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
   <PropertyGroup>
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
   </PropertyGroup>
   <ItemGroup>
     <Compile Remove="Logger\LoggerProvider.cs" />
+    <Compile Remove="Resources\RegressionType.cs" />
```

Comment:
I'm not sure why visual studio does this, but sometimes it flags files as removed, I typically look at the csproj file and remove these lines. No need for them to be listed here, that I know of, just bloat.